### ISSUE TITLE
can not create x-ray or wordwise for any book due to UnicodeDecodeError

### ISSUE THREAD

**Issue by littlezhanzhan** [xxyzz/WordDumb#124]
### Checkboxes

- [X] I have read the document at [xxyzz.github.io/WordDumb](https://xxyzz.github.io/WordDumb).
- [X] I have not found similar issue or disscussion at GitHub.
- [X] Reboot doesn't fix the problem.

### Describe the bug

if i choose use cuda, I can not create x-ray or wordwise for any book due to UnicodeDecodeError, I have restart more time. and the error msg is all the same ,"UnicodeDecodeError: 'utf-8' codec can't decode byte 0xce in position 2639: invalid continuation byte " 

### System Information

OS : windows 10 19045
Python 3.10
Built on Wed_Sep_21_10:41:10_Pacific_Daylight_Time_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0

pip info
Package      Version
------------ -------
blinker      1.6.2
click        8.1.3
colorama     0.4.6
Flask        2.3.2
itsdangerous 2.1.2
Jinja2       3.1.2
MarkupSafe   2.1.2
pip          22.3.1
rapidfuzz    3.0.0
setuptools   65.5.0
Werkzeug     2.3.3

### Error message

```text
calibre, version 6.13.0
错误： 未处理的异常: <b>UnicodeDecodeError</b>:'utf-8' codec can't decode byte 0xce in position 2639: invalid continuation byte

calibre 6.13  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: zh_CN
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Ebook Translator (1, 3, 5) && EpubSplit (3, 6, 0) && KindleUnpack - The Plugin (0, 83, 1) && Send to Kindle (1, 1, 0) && WordDumb (3, 29, 1)
Traceback (most recent call last):
  File "calibre\gui2\__init__.py", line 720, in dispatch
  File "calibre_plugins.worddumb.ui", line 144, in done
  File "calibre_plugins.worddumb.error_dialogs", line 43, in job_failed
  File "calibre_plugins.worddumb.error_dialogs", line 71, in subprocess_error
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xce in position 2639: invalid continuation byte
```

### Reproduce steps

1、choose any books
2、choose x-ray or wordwise with cuda

### Screenshots or videos

![Uploading image.png…]()

**Comment by littlezhanzhan** [comment]
### This is debug log

ApplicationPaletteChange event ignored
Using calibre Qt style: True
ApplicationPaletteChange event received
calibre 调试日志
calibre 6.13  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: zh_CN
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Ebook Translator (1, 3, 5) && EpubSplit (3, 6, 0) && KindleUnpack - The Plugin (0, 83, 1) && Send to Kindle (1, 1, 0) && WordDumb (3, 29, 1)
calibre 6.13  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: zh_CN
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Ebook Translator (1, 3, 5) && EpubSplit (3, 6, 0) && KindleUnpack - The Plugin (0, 83, 1) && Send to Kindle (1, 1, 0) && WordDumb (3, 29, 1)
QPA platform: windows
devicePixelRatio: 2.5
logicalDpi: 96.0 x 96.0
physicalDpi: 113.41395348837209 x 113.12164948453608
[0.00] Starting up...
[0.00] Showing splash screen...
[0.80] splash screen shown
[0.80] Initializing db...
[0.80] db initialized
[0.80] Constructing main UI...
Screens currently in system:
{'depth': 32,
 'device_pixel_ratio': 2.5,
 'geometry_in_logical_pixels': {'height': 864, 'width': 1536, 'x': 0, 'y': 0},
 'index_in_screens_list': 0,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY1',
 'serial': '',
 'size_in_logical_pixels': {'height': 864, 'width': 1536},
 'virtual_geometry': {'height': 864, 'width': 1536, 'x': 0, 'y': 0}}
Restoring geometry for: jobs_dialog_geometry
{'frame_geometry': {'height': 363, 'width': 650, 'x': 620, 'y': 97},
 'full_screened': False,
 'geometry': {'height': 334, 'width': 650, 'x': 620, 'y': 126},
 'maximized': False,
 'normal_geometry': {'height': 334, 'width': 650, 'x': 620, 'y': 126},
 'screen': {'depth': 32,
            'device_pixel_ratio': 2.5,
            'geometry_in_logical_pixels': {'height': 864,
                                           'width': 1536,
                                           'x': 0,
                                           'y': 0},
            'index_in_screens_list': 0,
            'manufacturer': '',
            'model': '',
            'name': '\\\\.\\DISPLAY1',
            'serial': '',
            'size_in_logical_pixels': {'height': 864, 'width': 1536},
            'virtual_geometry': {'height': 864, 'width': 1536, 'x': 0, 'y': 0}}}
Matching screen: {'depth': 32,
 'device_pixel_ratio': 2.5,
 'geometry_in_logical_pixels': {'height': 864, 'width': 1536, 'x': 0, 'y': 0},
 'index_in_screens_list': 0,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY1',
 'serial': '',
 'size_in_logical_pixels': {'height': 864, 'width': 1536},
 'virtual_geometry': {'height': 864, 'width': 1536, 'x': 0, 'y': 0}}
Setting widget geometry to: {'height': 334, 'width': 650, 'x': 620, 'y': 126}
Restoring geometry for: calibre_main_window_geometry
{'frame_geometry': {'height': 510, 'width': 1187, 'x': 55, 'y': 40},
 'full_screened': False,
 'geometry': {'height': 481, 'width': 1187, 'x': 55, 'y': 69},
 'maximized': False,
 'normal_geometry': {'height': 481, 'width': 1187, 'x': 55, 'y': 69},
 'screen': {'depth': 32,
            'device_pixel_ratio': 2.5,
            'geometry_in_logical_pixels': {'height': 864,
                                           'width': 1536,
                                           'x': 0,
                                           'y': 0},
            'index_in_screens_list': 0,
            'manufacturer': '',
            'model': '',
            'name': '\\\\.\\DISPLAY1',
            'serial': '',
            'size_in_logical_pixels': {'height': 864, 'width': 1536},
            'virtual_geometry': {'height': 864, 'width': 1536, 'x': 0, 'y': 0}}}
Matching screen: {'depth': 32,
 'device_pixel_ratio': 2.5,
 'geometry_in_logical_pixels': {'height': 864, 'width': 1536, 'x': 0, 'y': 0},
 'index_in_screens_list': 0,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY1',
 'serial': '',
 'size_in_logical_pixels': {'height': 864, 'width': 1536},
 'virtual_geometry': {'height': 864, 'width': 1536, 'x': 0, 'y': 0}}
Setting widget geometry to: {'height': 481, 'width': 1187, 'x': 55, 'y': 69}
[1.59] main UI initialized...
[1.59] Hiding splash screen
Starting QuickView
calibre 6.13  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: zh_CN
Successfully initialized third party plugins: DeDRM (10, 0, 3) && Ebook Translator (1, 3, 5) && EpubSplit (3, 6, 0) && KindleUnpack - The Plugin (0, 83, 1) && Send to Kindle (1, 1, 0) && WordDumb (3, 29, 1)
Traceback (most recent call last):
  File "calibre\gui2\__init__.py", line 720, in dispatch
  File "calibre_plugins.worddumb.ui", line 144, in done
  File "calibre_plugins.worddumb.error_dialogs", line 43, in job_failed
  File "calibre_plugins.worddumb.error_dialogs", line 71, in subprocess_error
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xce in position 2639: invalid continuation byte

**Comment by xxyzz** [comment]
Should be fixed in https://github.com/xxyzz/WordDumb/commit/84c5b0a1443eaa84762fda8923c39892376db308, please download test plugin from GitHub Actions Artifacts: https://github.com/xxyzz/WordDumb/actions/workflows/tests.yml

**Comment by littlezhanzhan** [comment]
> Should be fixed in [84c5b0a](https://github.com/xxyzz/WordDumb/commit/84c5b0a1443eaa84762fda8923c39892376db308), please download test plugin from GitHub Actions Artifacts: https://github.com/xxyzz/WordDumb/actions/workflows/tests.yml

yes,it help, thank you! and then , who used cuda should install the vs 2022 for c++ and the windows10 sdk at the same time
